pom.xml

```diff
@@ -376,7 +376,7 @@ under the License.
 			<dependency>
 				<groupId>org.apache.flink</groupId>
 				<artifactId>flink-shaded-netty</artifactId>
-				<version>4.1.90.Final-${flink.shaded.version}</version>
```

Comment:
The commit message doesn't match the change? :thinking:

Reply:
I'll change that when I'm rebasing